### ISSUE TITLE
Accessibility - Main Menu Steps - VoiceOver Reads Steps Twice

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/Ilab-do-good#48]
In the main menu VoiceOver reads the steps twice. Instead of the text and arrows being read as one item in the main menu. VoiceOver is reading them twice such as "step 1 step 1", "step2 step 2", etc. then "Button".

Similar to the show hide links in the Step Topic screens having text and only being able to activate the arrows the main menu should work similarly. Each Step and related arrow key should be read as one item.

Plus, as they are read the screen reader user is not notified that they are links or buttons.

**Comment by jfc3-dol** [comment]
@adriennerlong,

VoiceOver reads the Step and number. Then it reads the toggle button and says "Expand list of Topics" or "Collapse list of topics" depending on the buttons state. It would be easier for user to understand if VoiceOver read "Expand Step X list of Topics" or "Collapse Step X list of Topics" where "X' is the number of the step.

**Comment by adriennerlong** [comment]
From ICF: Fixed

**Comment by jfc3-dol** [comment]
VoiceOver now reads the first Step as "Step 5" and the second one as "Expand to list topics for step 5".

Going to remove the iOS label and wait to test this on Android before closing.

**Comment by jfc3-dol** [comment]
TalkBack now reads the first Step as "Step 1" and the second one as "Expand to list topics for Step 1".

Going to remove the TalkBack label and wait to test this on the web version before closing.

**Comment by jfc3-dol** [comment]
JAWS now reads the first Step as "Step 1" and the second one as "Expand to list topics for Step 1" or something like that.

Going to remove the web label and close the issue.
